_data/toc.yaml

```diff
@@ -1284,8 +1284,6 @@ manuals:
   - path: /engine/deprecated/
     title: Deprecated features
 
-  - path: /app/working-with-app/
```

Comment:
We should have a redirect to somewhere

There's also the reference docs (perhaps we need to keep them for now, but if we remove them, they need a redirect as well); https://github.com/docker/docker.github.io/tree/master/engine/reference/commandline

Reply:
This is fixed. Added a redirect and moved the content of the Working with Docker App topic to /engine/reference/commandline/app/ page.